### DISCUSSION TITLE
Welcome to Ludwig Discussions!

### DISCUSSION THREAD

**Discussion by w4nderlust** [ludwig-ai/ludwig#1107] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MineardAI** [comment]
Hi, my name is Tyrone and I started learning with Ludwig a little over a year ago. I chose Ludwig for the no programming feature and to get something extra back from Uber. I drive for them and felt I needed to learn some new skills just in case I have to go back to the corporate world. I will likely go down the rabbit hole further and take a data analytics and python course.

**Comment by skyemoor** [comment]
My name is Will and I'm investigating various AutoML, and focusing in this case on a project I'm doing in my spare time. I'll compare the output of Ludwig with 'manual' modeling efforts

**Reply by w4nderlust** [reply]
I suggest yu to consider specifically tasks that have big datasets and multimodal data, as those are the ones Ludwig is more suited for, while it is not particularly optimized for purely tabular datasets.

**Comment by byoungkyuji** [comment]
Hi, my name is byoungkyu and i'm using ludwig for making deep learning models.
I have learned ML, SW skills from making models and analyzing ludwig from a year ago, so maybe I can help fixing some bugs or testing.
thank you.

**Reply by w4nderlust** [reply]
You are more than welcome to contribute! If you want to help fixing bugs, there are a few open ones in the issues tab, we can also help directing you to good introductory ones!

**Comment by why-does-ie-still-exist** [comment]
I'm Michael and I'm toying around with machine learning. About a year ago I tried to set up CUDA to do machine learning on my local system, only to give up 6ish hours later due to all the complications(anaconda has issues on windows, CUDA has weird dependencies like VS 2017, and a million other tiny bugs). Recently I found out about [colab](https://colab.research.google.com), so I've started using it with Ludwig. I have very basic python skills but I'm pretty good with a terminal so I thought Ludwig would be a great place to start experimenting. I have run into a lot of issues a more experienced ML engineer might not even think of (but are obvious to a beginner) so I might might start making some feature requests (the biggest issue I've seen so far is that Ludwig should crash more gracefully in certain situations).

**Reply by w4nderlust** [reply]
Sure, feel free to post the issues you run into and we'll try to help!
